wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -1550,14 +1550,41 @@ private function execute_describe() {
 	 */
 	private function execute_update() {
 		$this->rewriter->consume(); // Update.
-
-		$params = array();
+		$limit    = $this->rewriter->peek(
+			array(
+				'type'  => WP_SQLite_Token::TYPE_KEYWORD,
+				'value' => 'LIMIT',
+			)
+		);
+		$order_by = $this->rewriter->peek(
+			array(
+				'type'  => WP_SQLite_Token::TYPE_KEYWORD,
+				'value' => 'ORDER BY',
+			)
+		);
+		$where    = $this->rewriter->peek(
+			array(
+				'type'  => WP_SQLite_Token::TYPE_KEYWORD,
+				'value' => 'WHERE',
+			)
+		);
+		$params   = array();
 		while ( true ) {
 			$token = $this->rewriter->peek();
 			if ( ! $token ) {
 				break;
 			}
 
+			if ( $token->value === 'WHERE' && ( $limit || $order_by ) ) {
```

Comment:
Let's add a block comment to explain how this is needed to support LIMIT and ORDER BY clauses.